concepts/05 UI Components/Toolbar/00 Getting Started with Toolbar/20 Create an Overflow Menu.md

```diff
@@ -0,0 +1,302 @@
+The Toolbar can render its items in the overflow menu. Specify the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property for an item. Use one of the following values:
+
+- *"always"*  
+ Always places the item in the overflow menu. You can specify the order of such items in the overflow menu.
+
+- *"never"* (default)  
+ Places the item outside of the overflow menu.
+
+- *"auto"*  
+ Places the item outside of the overflow menu. If all items cannot fit within the [width](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#width) of the Toolbar, it renders this item in the overflow menu. The Toolbar component determines the order of such items in the overflow menu automatically.
+
+The previous section of the tutorial describes how to customize Toolbar items. If you use an [itemTemplate](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#itemTemplate) to customize items in the Toolbar with `locateInMenu="auto"` and you want to keep the customization if the items get into the overflow menu, specify the [menuItemTemplate](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#menuItemTemplate). You can also use this property to customize the overflow menu item appearance.
```

Comment:
Now that I re-read this, I'm not sure I understand what you're saying. 

So an item's `locateInMenu` can be `always` or `auto`. Otherwise it never appears in the overflow menu. You're saying that if the property value is `auto`, then I should use `menuItemTemplate`. But what if it's `always`? 

Shouldn't this paragraphs just say "If you want to apply a template to an item in the overflow menu, specify `menuItemTemplate`". I suspect that it doesn't depend on whether it's `auto` or `always`.